src/main/java/io/kestra/plugin/aws/eventbridge/PutEvents.java

```diff
@@ -0,0 +1,157 @@
+import software.amazon.awssdk.services.eventbridge.model.PutEventsResponse;
+
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send multiple custom events as maps to Amazon EventBridge so that they can be matched to rules",
+            code = {
+                "entries:",
+                "  - eventBusName: \"events\"",
+                "    source: \"Kestra\"",
+                "    detailType: \"my_object\"",
+                "    detail:",
+                "      message: \"hello from EventBridge and Kestra\"",
+            }
+        ),
+        @Example(
+            title = "Send multiple custom events as a JSON string to Amazon EventBridge so that they can be matched to rules.",
+            code = {
+                "entries:",
+                "  - eventBusName: \"events\"",
+                "    source: \"Kestra\"",
+                "    detailType: \"my_object\"",
+                "    detail: \"{\\\"message\\\": \\\"hello from EventBridge and Kestra\\\"}\"",
+                "    resources:",
+                "      - \"arn:aws:iam::123456789012:user/johndoe\"",
+            }
+        )
+    }
+)
+@Schema(
+    title = "Send multiple custom events to Amazon EventBridge so that they can be matched to rules."
+)
+public class PutEvents extends AbstractConnection implements RunnableTask<PutEvents.Output> {
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @Schema(
+        title = "List of event entries to send to EventBridge",
+        description = "A list of at least one EventBridge entry."
+    )
+    private List<Entry> entries;
+
+    @Override
+    public PutEvents.Output run(RunContext runContext) throws Exception {
+        final long start = System.nanoTime();
+
+        try (var eventBridgeClient = client(runContext)) {
+            List<PutEventsRequestEntry> requestEntries = new ArrayList<>(entries.size());
+
+            for (Entry entry : entries) {
+                requestEntries.add(entry.toRequestEntry(runContext));
+            }
+
+            PutEventsRequest eventsRequest = PutEventsRequest.builder()
+                .entries(requestEntries)
+                .build();
+
+            PutEventsResponse putEventsResponse = eventBridgeClient.putEvents(eventsRequest);
+
+            runContext.metric(Timer.of("duration", Duration.ofNanos(System.nanoTime() - start)));
+            runContext.metric(Counter.of("failedEntryCount", putEventsResponse.failedEntryCount()));
+            runContext.metric(Counter.of("successfulEntryCount", entries.size() - putEventsResponse.failedEntryCount()));
+            runContext.metric(Counter.of("entryCount", entries.size()));
+
+            List<OutputEntry> outputEntries = new ArrayList<>(putEventsResponse.entries().size());
```

Comment:
Don't think it's a good idea to load every response as output will eventId, @anna-geller WDYT ?

Reply:
why not? having eventId somewhere in the logs or outputs seems super helpful for troubleshooting 

do you have a preference for an alternative option to share that information e.g. via logs?

Reply:
to use internal storage and not output.
Output are here only for orchestration logic

Reply:
yup, makes sense to send the output as an ION file rather than plain response text 👍 especially when someone would send hundreds of entries. should have thought about it, sorry

Reply:
@moadibfr can you change the output to use internal storage? makes sense, especially for larger responses